File: lootboxesAreGambling/java/src/main/java/Dice.java
public class Dice {

    private Integer Faces;

    private Boolean IsLoaded;

    public Dice(Integer faces, Boolean isLoaded)
    {
        Faces = faces;
        IsLoaded = isLoaded;
    }

    public Integer getFaces() {
        return Faces;
    }

    public void setFaces(Integer faces) {
        Faces = faces;
    }

    public Boolean getIsLoaded() {
        return IsLoaded;
    }

    public void setisLoaded(Boolean isLoaded) {
        IsLoaded = isLoaded;
    }
}

File: lootboxesAreGambling/netCore/Models/Dice.cs
namespace netCore.Models
{
    using System;
    using System.Collections.Generic;

    public class Dice
    {
        public int Faces {get; set;}

        public bool IsLoaded {get; set;}

        public Dice()
        {
        }

        public Dice(int faces, Boolean isLoaded)
        {
            Faces = faces;
            IsLoaded = isLoaded;
        }
    }
}